gen/generate_test.go

```diff
@@ -100,3 +103,22 @@ func TestLoadSingleRootKeyConcurrent(t *testing.T) {
 	}
 	wg.Wait()
 }
+
+func TestGenesisRoundoff(t *testing.T) {
+	defer func() {
+		verboseOutWriter = os.Stdout
```

Comment:
I like the idea of having the ability to redirect output during tests. However, this would cause race conditions.
Maybe you could pass the output file writer as an argument to the `GenerateGenesisFiles `/ `setupGenerateGenesisFiles` method instead ? i.e. instead of passing the `verbose` flag, pass in this *os.File target.

Reply:
ok, I like the idea of an io.Writer as the `verbose` flag instead of a bool. changing...